File: 프로그래머스/lv0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    is_id_match = False
    id = id_pw[0]
    pw = id_pw[1]
    for i in db:
        if id == i[0]:
            is_id_match = True
            if pw == i[1]:
                return "login"
    if is_id_match:
        return "wrong pw"
    return "fail"

File: 프로그래머스/lv0/120883. 로그인 성공？/로그인 성공？.go
func solution(id_pw []string, db [][]string) string {
    isIdMatch := false
    for _, e := range db {
        if id_pw[0] == e[0] {
            isIdMatch = true
            if id_pw[1] == e[1] {
                return "login"
            }
        }
    }
    if isIdMatch {
        return "wrong pw"
    }
    return "fail"
}